style.css

```
@@ -0,0 +1,147 @@
+body {
+  font-family: "Monserrat", sans-serif;
+  /* overflow: hidden; */
+  background-image: url("assets/bg2.png");
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: cover;
+  background-attachment: fixed;
+}
+:root {
+  --primary-colour: ;
+  --secondary-colour: ;
+  --text-colour: ;
+}
+
+form {
+  background: black;
+  text-align: center;
+}
+section {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  padding-left: 50px;
```

Try keep padding for sections symetrical.
```suggestion
  padding: 0 50px;
```